elastic/apm-server
# Copy `error.log.message` or `error.exception.message` to `message`

## Motivation/summary

Copy `error.log.message` or `error.exception.message` to the ECS field `message`. Add `message` to error fields.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. Send errors to apm-server
2. Check that the `message` field is populated
3. Do this above with `apm-server.data_streams.enabled`, and then navigate to the Logs app and check that errors show up

## Related issues

Closes https://github.com/elastic/apm-server/issues/5971
